lib/Watching.js

```diff
@@ -168,6 +168,7 @@ class Watching {
 	close(callback) {
 		if (callback === undefined) callback = () => {};
 
+		this.compiler.running = false;
```

Comment:
This might be too early. A compilation can still be running. Could you move this later in this code, just before the `callback()` calls.